### PR TITLE
Adding ordinal numbers to the dropped elements

### PR DESCRIPTION
**Pull request by romanblanco**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1514117

### CODE REVIEW

File: src/dialog-editor/components/box/box.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
             <div class="well well-lg nosort"
                  ng-if='box.dialog_fields.length === 0'>
               <i class="fa fa-object-group"></i>
-              {{ 'Drag your components here' | translate }}
+              {{ 'Drag items here to add to dialog. At least one item is required before saving' | translate }}
```
Comment (martinpovolny) [inline src/dialog-editor/components/box/box.html]: this does not sound very English ;-)

Comment (romanblanco) [reply src/dialog-editor/components/box/box.html]: @martinpovolny thanks, corrected.

**Comment by romanblanco** [comment]
@miq-bot add_label gaprindashvili/yes bug

**Comment by romanblanco** [comment]
@karelhala or @himdel, can you review?

**Comment by romanblanco** [comment]
@miq-bot add_label gaprindashvili/yes
@miq-bot add_label bug

**Comment by romanblanco** [comment]
Failing Travis should be fixed by https://github.com/ManageIQ/ui-components/pull/211

**Comment by martinpovolny** [comment]
Restarting travis.

**Comment by himdel** [comment]
@miq-bot add_label gaprindashvili/backported
@miq-bot remove_label gaprindashvili/yes

This was released in manageiq-ui-components 1.0.2, backported to gaprindashvili via https://github.com/ManageIQ/manageiq-ui-classic/pull/2951.

**Pull request merged**